Automatic Validation ended with:
> 2024-03-06T21:00:46.7050807Z ##[error] One or more errors occurred. (Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\a\amrdeveloper\fileql\0.2.0', expected 'manifests\a\amrdeveloper\fileql\0.7.1') Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\a\amrdeveloper\fileql\0.2.0', expected 'manifests\a\amrdeveloper\fileql\0.7.1'

(Automated response - build 867.)

--- a/manifests/a/amrdeveloper/fileql/0.2.0/amrdeveloper.fileql.installer.yaml
+++ b/manifests/a/amrdeveloper/fileql/0.2.0/amrdeveloper.fileql.installer.yaml
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
 
 PackageIdentifier: amrdeveloper.fileql
-PackageVersion: 0.7.1
+PackageVersion: 0.2.0
 InstallerType: portable
 Commands:
 - gitql

--- a/manifests/a/amrdeveloper/fileql/0.2.0/amrdeveloper.fileql.installer.yaml
+++ b/manifests/a/amrdeveloper/fileql/0.2.0/amrdeveloper.fileql.installer.yaml
@@ -5,7 +5,7 @@ PackageIdentifier: amrdeveloper.fileql
 PackageVersion: 0.2.0
 InstallerType: portable
 Commands:
-- gitql
+- fileql
 Installers:
 - Architecture: x86
   InstallerUrl: https://github.com/AmrDeveloper/FileQL/releases/download/0.2.0/fileql-x86-windows.exe

--- a/manifests/a/amrdeveloper/fileql/0.2.0/amrdeveloper.fileql.yaml
+++ b/manifests/a/amrdeveloper/fileql/0.2.0/amrdeveloper.fileql.yaml
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
 
 PackageIdentifier: amrdeveloper.fileql
-PackageVersion: 0.2.1
+PackageVersion: 0.2.0
 DefaultLocale: en-US
 ManifestType: version
 ManifestVersion: 1.5.0

